Assets/Mirage/Editor/WelcomeWindow/WelcomeWindow.cs

```diff
@@ -229,6 +238,67 @@ private void ToggleMenuButtonColor(Button button, bool toggle)
             }
         }
 
+        //parse the change log file
+        private List<string> ParseChangeLog()
+        {
+            List<string> content = new List<string>();
+
+            using(StreamReader reader = new StreamReader(changeLogPath))
+            {
+                string line;
+
+                while((line = reader.ReadLine()) != null)
+                {
+                    //we dont need to parse an empty string
+                    if(line == string.Empty) { continue; }
+
+                    //always add the first line
+                    if(content.Count == 0) { content.Add(line); continue; }
```

Comment:
one statement per line so that I can use breakpoints

```suggestion
                    if(content.Count == 0) 
                    {
                        content.Add(line); 
                        continue;
                    }
```